package.json

```diff
@@ -106,7 +106,7 @@
     "@binary-com/binary-document-uploader": "^2.4.4",
     "@binary-com/binary-style": "^0.2.26",
     "@deriv-com/analytics": "^1.26.1",
-    "@deriv-com/auth-client": "^1.3.10",
+    "@deriv-com/auth-client": "^1.4.3",
```

Comment:
Updated auth-client to automatically include `id_token_hint` when logging out, which will help invoke the correct front channels